src/Data/Expr.purs

```diff
@@ -107,7 +109,7 @@ print :: ExprType -> String
 print = case _ of
   Array x -> "[" <> intercalate ", " (map print x) <> "]"
   Boolean x -> show x
-  Cents x -> formatCentsToStrDollars x
+  Cents x -> replace (Pattern ".00") (Replacement "") (formatCentsToStrDollars x)
```

Comment:
This arguably should be handled in Ocelot, but didn't want to tag another release just for this small feature. This prevents rounded dollar amounts in the `Currency` fields from being formatted with an extra ".00"

Reply:
Gotcha! More than happy to merge this now and move to `ocelot` later.